docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,410 @@
+These instructions are common for the docker and without docker, so please skip the next section and jump to
+"Initialising the web app"
+
+## Setup without Docker
+
+This is the traditional python-django route that we will follow. Make sure PostgreSQL, SOLR and Memcached(optional) have
+been setup as described above. Python3 is recommended.
+
+Start by cloning the repository:
+```bash
+git clone https://github.com/macports/macports-webapp.git
+cd macports-webapp
+```
+
+**NOTE**: It is recommended that you clone your fork for development.
+
+Now, create a virtual environment and activate it.
+
+```bash
+python3 -m venv env
+source env/bin/activate
+```
+
+This will create an `env` directory in the root of the project.
+
+Install the requirements for our Django app:
+
+```
+pip install -r app/requirements.txt
+```
+
+Once all the dependencies are installed we can move to the most important step, that is connecting various services
+to the django app. If you followed the setup with Docker, you already know that we do this by using environment variables.
+To set the environment variables inside your virtual env, run the following commands:
+
+```bash
+export DB_NAME=                            (name of our database, webapp)
+export DB_USER=                            (username of the database user)
+export DB_PASSWORD=                        (password for your database user)
+export DB_HOST=                            (localhost)
+export DB_PORT=                            (default is 5432)
+export SECRET_KEY=                         (any string, but secret)
+export SOLR_URL=                           (http://127.0.0.1:8983/solr/tester)
+export SOLR_ADMIN_URL=                     (http://127.0.0.1:8983/solr/admin/cores)
+export EMAIL_HOST=                         (smtp host, optional)
+export EMAIL_PORT=                         (smtp port, optional)
+export EMAIL_HOST_USER=                    (smtp user, optional)
+export EMAIL_HOST_PASSWORD=                (smtp user password, optional)
+```
+
+**NOTE**: Add the values to the keys as explained and then remove the command. Anything between the `()` is just for your
+reference and should not be run with the command. A sample command would look like:
+
+```bash
+export DB_NAME=webapp
+```
+
+Setting up env variables every time is a tedious task and you might want to reduce the friction by following some really
+good guides: https://help.pythonanywhere.com/pages/environment-variables-for-web-apps/
+
+Once the database (and other services) have been connected, run the migrations. Before that let's cd into the main src 
+directory for the project, that is `app`.
+
+```bash
+cd app
+python3 manage.py migrate
+python3 manage.py collectstatic
+```
+
+You may now run the server, but your app has no data, yet.
+
+```bash
+python3 manage.py runserver
+```
+
+## Initialising the Webapp
+
+Now that you have the app running(using either of two routes above), it is time to add some data to the app. Some of these
```

Comment:
This applies to a couple of more places. Opening parenthesis should almost always be preceded by whitespace.  
```suggestion
Now that you have the app running (using either of two routes above), it is time to add some data to the app. Some of these
```